internal/server/auth/method/token/server.go

```diff
@@ -37,20 +38,32 @@ func (s *Server) RegisterGRPC(server *grpc.Server) {
 	auth.RegisterAuthenticationMethodTokenServiceServer(server, s)
 }
 
+func (s *Server) SkipsAuthentication(ctx context.Context) bool {
+	return true
+}
```

Comment:
The token service never skipped authentication. Otherwise, anyone can make a token.

It does however need to force an error when namespaced tokens are presented, as discussed.
```suggestion
func (s *Server) AllowsNamespaceScopedAuthentication((ctx context.Context) bool {
	return false
}
```

Reply:
Note: the other methods need to skip auth, because they all do their own auth separately (i.e. delegated to IdP and validating JWTs).